.travis.yml

```diff
@@ -8,4 +8,5 @@ services:
 - rabbitmq
 script: npm test
 before_install:
-- openssl aes-256-cbc -K $encrypted_c76f7cd53afb_key -iv $encrypted_c76f7cd53afb_iv -in user-config.yml.enc -out user-config.yml -d
+# unpack the secrets, but only if this is not a pull request; otherwise, the tests will run with no credentials
+- '[ "${TRAVIS_PULL_REQUEST}" = "false" ] && openssl aes-256-cbc -K $encrypted_c76f7cd53afb_key -iv $encrypted_c76f7cd53afb_iv -in user-config.yml.enc -out user-config.yml -d
```

Comment:
It's easier to just set env vars on travis... it has a UI for doing this now... then we don't have ugly things in .travis... anyways just a note... don't have to do it here it...just saying...